### PR TITLE
Fix merge issues

### PR DESCRIPTION
**Pull request by Priscilla-AL**
What does this PR do?
Fixes analytics-netmanager merge styling issues

How do I test it out?
cd netmanager
npm install
npm run prod-pc
checking out the appearance of the dashboard. Map, graphs, cards

### CODE REVIEW

File: netmanager/src/views/pages/Dashboard/Dashboard.js [hunk 1]
```diff
@@ -525,62 +530,12 @@ const Dashboard = (props) => {
...
         <Grid item lg={6} md={6} sm={12} xl={6} xs={12}>
           <Grid item lg={12} sm={12} xl={12} xs={12}>
-            <Map id="rootMapContainerId" />
+          <div className={classes.mapContainer}>
+              <Map/>
+            </div>
           </Grid>
 
-          {/* <Grid container spacing={0} className="MapCardContent">
-            <Grid item lg={2} sm={4} xl={2} xs={12}>
```
Comment (Baalmart) [inline netmanager/src/views/pages/Dashboard/Dashboard.js:532]: Just not sure about this huge comment/deletion but all looks good on running!

**Review by Baalmart — APPROVED** [review]
Works as expected.

File: netmanager/src/views/pages/Dashboard/Dashboard.js (hunk 1 above)
Comment (Priscilla-AL) [reply netmanager/src/views/pages/Dashboard/Dashboard.js:532]: That's the old legend bar that was bottom of the map, was replaced with a smaller legend floating on the map. 
Was initially commented out by me.

Comment (Baalmart) [reply netmanager/src/views/pages/Dashboard/Dashboard.js:532]: Okay, awesome.

**Pull request merged**